insights/specs/datasources/lpstat.py

```diff
@@ -0,0 +1,44 @@
+"""
+Custom datasources for lpstat information
+"""
+from insights.core.context import HostContext
+from insights.core.dr import SkipComponent
+from insights.core.plugins import datasource
+from insights.core.spec_factory import DatasourceProvider, simple_command
+from insights.specs import Specs
+
+
+class LocalSpecs(Specs):
+    """ Local specs used only by lpstat datasources """
+
+    lpstat_v = simple_command("/usr/bin/lpstat -v")
+    """ Returns the output of command ``/usr/bin/lpstat -v`` """
+
+
+@datasource(LocalSpecs.lpstat_v, HostContext)
+def lpstat_protocol_printers_info(broker):
+    """
+    This datasource provides the not-sensitive information collected
+    from ``/usr/bin/lpstat -v``.
+
+    Typical content of ``/usr/bin/lpstat -v`` file is::
+
+        "device for NAY_10F_Smurfs: ipp://cups.nay.redhat.com/printers/NAY_10F_Smurfs"
+
+    Returns:
+        dict: string containing non-sensitive information.
```

Comment:
@wushiqinlou Could you update this to say that it returns a DatasourceProvider similar to how the package_provides datasource is documented?

Reply:
@ryan-blakley - Updated, please have a check.

Reply:
@wushiqinlou Thanks it looks good to me.